contracts/src/v0.8/dev/KeeperRegistryLogic.sol

```diff
@@ -8,15 +8,19 @@ import "./KeeperRegistryBase.sol";
  */
 contract KeeperRegistryLogic is KeeperRegistryBase {
   /**
+   * @param paymentModel one of Default, Arbitrum, Optimism
+   * @param registryGasOverhead the registry gas overhead
    * @param link address of the LINK Token
    * @param linkEthFeed address of the LINK/ETH price feed
    * @param fastGasFeed address of the Fast Gas price feed
    */
   constructor(
+    uint8 paymentModel,
```

Comment:
nit. can you use the enum name as type here?

Reply:
Done